Plugin/OrderRepositoryPlugin.php

```diff
@@ -10,8 +10,8 @@
 
 class OrderRepositoryPlugin
 {
-    /** @var OrderRelatedDataRepositoryInterface */
-    private $orderRelatedDataRepository;
+    public \Magento\Framework\Api\ExtensionAttributesFactory $extensionAttributesFactory;
+    private \SnowIO\ExtendedSalesRepositories\Api\OrderRelatedDataRepositoryInterface $orderRelatedDataRepository;
```

Comment:
Just seen this https://github.com/snowio/magento2-extended-sales-repositories/blob/c2939797952acd20336ae744178386287fd77aec/Plugin/OrderRepositoryPlugin.php#L52 ... `$orderExtensionFactory` is not in the constructor... This a disaster waiting to happen! I guess an order always having extension attributes in most situations has saved us a bit here. Might be worth configuring PHPStan of PHPCS for some of these modules